packages/gluon/gluon.0.0.9/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis:
+  "A minimal, portable, and fast API on top of the operating-system's evented I/O API"
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+tags: ["async" "io" "tcp" "udp" "multicore"]
+homepage: "https://github.com/riot-ml/gluon"
+bug-reports: "https://github.com/riot-ml/gluon/issues"
+depends: [
+  "bytestring" {>= "0.0.1"}
+  "config" {>= "0.0.1"}
+  "libc" {>= "0.0.1"}
+  "ocaml" {>= "5.1"}
+  "rio" {>= "0.0.1"}
+  "uri" {>= "4.4.0"}
+  "dune" {>= "3.11"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/riot-ml/gluon.git"
+available: arch != "x86_32" & arch != "arm32" & arch != "ppc64" & arch != "s390x" & os != "freebsd"
```

Comment:
```suggestion
```
Then let's do this, so that an update of libc may enable this library to work and if not, the revdeps tests will point it out and I can add the necessary bounds later